modules/govuk_postgresql/manifests/backup.pp

```diff
@@ -34,4 +34,7 @@
         source => 'puppet:///modules/govuk_postgresql/etc/postgresql-backup-pre',
     }
 
+    # Include offsite backups to S3
+    include wal_e::backup
```

Comment:
Would it be better to explicitly state `govuk_postgresql::wal_e::backup` or actually just move it into it's entirely own `govuk_wale` module? I agree though, it's reads confusingly.